src/client/containers/PullRequestSummary/ReviewSection/ReviewerList/ReviewerList.css

```diff
@@ -0,0 +1,15 @@
+.APPROVED {
```

Comment:
CSS class naming conventions stand for dash-separated class names in lower case.